Doc/library/crypt.rst

```diff
@@ -41,17 +41,24 @@ are available on all platforms):
 .. data:: METHOD_SHA512
 
    A Modular Crypt Format method with 16 character salt and 86 character
-   hash.  This is the strongest method.
+   hash based on the SHA-512 hash function.  This is the strongest method.
 
 .. data:: METHOD_SHA256
 
    Another Modular Crypt Format method with 16 character salt and 43
-   character hash.
+   character hash based on the SHA-256 hash function.
+
+.. data:: METHOD_BLF
```

Comment:
Why not `METHOD_BLOWFISH`?

Reply:
FreeBSD provides an API for getting a list of supported methods. Their names are: "md5", "blf", "nth", "sha256", "sha512" and "des".

Reply:
Do we care about the FreeBSD API? We don't expose it...

Reply:
I don't have any arguments for other names.

Reply:
Well, my point is that `METHOD_BLOWFISH` is obvious while `METHOD_BLF` is obscure (except perhaps for someone familiar with FreeBSD APIs :-)).

Reply:
Okay, I'll change the name. In any time we can add an alias if it is needed.

Reply:
I prefer METHOD_BLOWFISH. I agree with @pitrou, Python is not FreeBSD.